WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/AddCustomAmountViewModel.swift

```diff
@@ -8,22 +8,55 @@ typealias CustomAmountEntered = (_ amount: String, _ name: String, _ feeID: Int6
 
 final class AddCustomAmountViewModel: ObservableObject {
     let formattableAmountTextFieldViewModel: FormattableAmountTextFieldViewModel
+    private var baseAmountForPercentage: Decimal
     private let onCustomAmountEntered: CustomAmountEntered
     private let analytics: Analytics
+    private let currencyFormatter: CurrencyFormatter
 
-    init(locale: Locale = Locale.autoupdatingCurrent,
+    var baseAmountForPercentageString: String {
+        guard let baseAmountForPercentageString = currencyFormatter.formatAmount(baseAmountForPercentage) else {
+            return ""
+        }
+
+        return "(\(baseAmountForPercentageString))"
```

Comment:
The return type is already a String, we don't need to interpolate it:

```suggestion
        return baseAmountForPercentageString
```

Reply:
Thanks for the point! Done in 05809022a584692dd4ccced04da47b0bb9964e79

Reply:
On a second look, I wanted to enclose the string on parenthesis, and thus the interpolation. I reversed that commit